src/main/java/org/terasology/Lakes/LavaProvider.java

```diff
@@ -62,35 +66,55 @@ public void process(GeneratingRegion region) {
 
         if (enabled) {
 
-            for (Vector3i pos : processRegion) {
+            Vector3i min = processRegion.min();
+            int STEP = 3;
```

Comment:
Did you mean to remove this in favor of the `SKIP_BLOCKS` constant? Which is unused. I'm going to figure as much and just update `STEP` to use that instead 👍 Everything else seems fine :-)